format/Schema.fbs

```diff
@@ -37,7 +37,7 @@ enum MetadataVersion:short {
   /// >= 0.8.0 (December 2017). Non-backwards compatible with V3.
   V4,
 
-  /// >= 1.0.0 (July 2020. Backwards compatible with V4 (V5 readers can read V4
+  /// >= 1.0.0 (July 2020). Backwards compatible with V4 (V5 readers can read V4
```

Comment:
there is a enum defined with "features" we should add column bag to that enum. Unfortunately, I haven't had the time to integrate the enum but the purpose was to allow clients to express which capabilities they handle the server.